test/HelperPluginManagerCompatibilityTest.php

```diff
@@ -58,7 +57,7 @@ protected function getV2InvalidPluginException()
 
     protected function getInstanceOf()
     {
-        return HelperInterface::class;
+        return 'callable';
```

Comment:
Of course, removing the `$instanceOf` property might affect this test. I'll try it out and see.